src/shared/extension/utils.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  ExtensionSettingsStorageKey,
+  extensionSettingsStorageKeys,
+} from './extension-settings-manager';
+
+const initiallyDisabledExtensionSettingsStorageKeys: ExtensionSettingsStorageKey[] =
+  [] as const;
```

Comment:
maybe we can just hardcode `true` few lines below in `createInitialExtensionSettingsStorageKeys`?

Reply:
sure, I just wanted to make it easier for the future, as I've heard that there can be some options that we would like to have initially disabled.

Reply:
let me know if we want to keep that for the future, or just remove it for now

Reply:
I am with with both 👍🏻 you decide